charts/library/common/templates/classes/_statefulset.tpl

```diff
@@ -42,6 +42,7 @@ spec:
   {{- else }}
   replicas: 1
   {{- end }}
+  minReadySeconds: {{ dig "statefulset" "minReadySeconds" 0 $statefulsetObject }}
```

Comment:
Is `0` the default value? Using a `with` statement should allow for it to not be set if it isn't in the values.